client/modules/IDE/components/EditableInput.jsx

```diff
@@ -58,7 +59,7 @@ function EditableInput({
       <button
         className="editable-input__label"
         onClick={beginEditing}
-        aria-label={`Edit ${displayValue} value`}
+        aria-label={this.props.t('EditableInput.EditValue', { display: displayValue })}
```

Comment:
The variable will be named `display` but the translation text in english uses `{{ displayValue }}` and in spanish `{{ display }}`. Please fix.

Reply:
yes, it should be ```display``` in both. It's fixed now.